biscuit-auth/src/token/mod.rs

```diff
@@ -73,9 +73,9 @@ pub fn default_symbol_table() -> SymbolTable {
 #[derive(Clone, Debug)]
 pub struct Biscuit {
     pub(crate) root_key_id: Option<u32>,
-    pub(crate) authority: schema::Block,
-    pub(crate) blocks: Vec<schema::Block>,
-    pub(crate) symbols: SymbolTable,
+    pub authority: schema::Block,
```

Comment:
not really comfortable with exposing this, it should not be necessary. We'll have to adapt the C API

Reply:
Addressed here: [`a71b2fb` (#234)](https://github.com/biscuit-auth/biscuit-rust/pull/234/commits/a71b2fb9413af1b07f176a5817785da32320b46a)

Thanks for the heads-up!